.github/workflows/sync-opencraft-release-palm-1.yml

```diff
@@ -0,0 +1,50 @@
+name: Sync opencraft-release/palm.1 with upstream
+
+# Controls when the workflow will run
+on:
+# TODO: Uncomment this out when its ready to be merged
+#   schedule:
+#     - cron: '0 0 * * 1'  # Runs every Monday 00:00 UTC
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    env:
+      # TODO: set this to the OpenCraft Requirements Bot PAT
+      GITHUB_TOKEN: ${{ secrets.SYNC_GH_ACTIONS_PAT }}
+      # The fork repo provided needs to be in the {owner}/{repo} format
+      FORK_REPO: open-craft/edx-platform
```

Comment:
oh cool! I wasn't familiar with `strategy` and `matrix`. I've updated the PR to include them, it makes things much cleaner. I also refactored the naming of the GH Action to make it more generic now that it can support arbitrary repos/branches